src/map/skill.cpp

```diff
@@ -11708,15 +11708,10 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 			if( !status_isdead(bl) )
 				break;
 
-			status_zap(bl, 0, tstatus->sp * 10 * skill_lv / 100);
-
-			int heal = tstatus->sp;
-
-			if( heal <= 0 )
-				heal = 1;
-			tstatus->hp = heal;
+			tstatus->hp = max(tstatus->sp, 1);
+			tstatus->sp = tstatus->sp * 10 * skill_lv / 100;
```

Comment:
I just saw that the formula is incorrect since https://github.com/rathena/rathena/commit/4b32621c61bfce397488e27b4eb962b7c04a9c6f
https://www.divine-pride.net/database/skill/2421
```
[Level 1] : 성공율 100% / 부활 대상의 SP 50% 소모
[Level 2] : 성공율 100% / 부활 대상의 SP 40% 소모
[Level 3] : 성공율 100% / 부활 대상의 SP 30% 소모
[Level 4] : 성공율 100% / 부활 대상의 SP 20% 소모
[Level 5] : 성공율 100% / 부활 대상의 SP 10% 소모 
```
It should consume 50% at level 1, 40% at level 2.. with the current formula the sp are 10% of the previous sp (90% sp consumed) at level 1 etc..

Reply:
My fault... :(